### PR TITLE
update README.md

### LINKED ISSUE #99: README.md: Update grammar at line 97
Under `Phase 4`, it says:
`For avoid these influences causing incorrect design decisions in coAST ...`

`For avoid` doesn't look quite right and hence is to be updated.

A good `newcomer` issue.

### PR DESCRIPTION
**Pull request by Marco2018**
README.md: Fix docstring typo

avoid -->avoiding

Closes https://github.com/coala/coAST/issues/99

### CODE REVIEW

**Comment by budukhyash** [comment]
@Marco2018  Hello please go through the newcomers guide https://api.coala.io/en/latest/Developers/Newcomers_Guide.html.At coala we dont submit a PR unless the issue is assigned to us.Contact the maintainers so that they assigne you the issue.

**Review by shashank-b — CHANGES_REQUESTED** [review]
Hey @Marco2018
The change looks fines, but I'd suggest you do the following things:

- Go through the [commit guidelines](http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html).
- Go through the [newcomer's guide](https://coala.io/newcomer).
- After reading them, squash your commits.
- Modify your commit message as per the other examples given in the guidelines.
- Refer the issue in the PR description (its hard to follow the trail otherwise).
- Be sure to not make the changes from your master branch later on (it can lead to some complications). You don't need to change this for this PR though.
- Make all the changes in the current PR and don't create another one.

After all this is done, someone would merge your PR :slightly_smiling_face:

**Comment by Marco2018** [comment]
Hi @shashank-b
Thank you for your suggestions. It is my first time to contribute in an open source project. I will go through commit guidelines and newcomer's guide and modify it afterward. Thanks!

**Comment by Marco2018** [comment]
Hi @shashank-b
I have read both commit guidelines and newcomer's guide and modified my commit message as you suggested. But I am not sure how to squash my commits. Can I just fork this project to my Github once again?
Thanks, it's my first time to make a PR. I really appreciate your help.

**Comment by KVGarg** [comment]
@Marco2018 Check [coala git basics](https://api.coala.io/en/latest/Developers/Git_Basics.html#squashing-your-commits) to know how can squashing of commits be done!

**Review by KVGarg — CHANGES_REQUESTED** [review]
@Marco2018 Check [coala git basics](https://api.coala.io/en/latest/Developers/Git_Basics.html#squashing-your-commits) to know how can squashing of commits be done!

**Comment by Marco2018** [comment]
Hey! @KVGarg Thank you for your help. I have tried to squash my commits into one. So I reset back the branch and commit once again.  It seems commits are squashed into one commit. Is it acceptable for this PR? So sorry I made this  PR a little bit complicated.  THX!

**Comment by KVGarg** [comment]
Yeah @Marco2018 
For further any changes to existing commit you can use `git commit —amend` to avoid creating more than one commit for a PR/issue

**Comment by Marco2018** [comment]
@KVGarg  Thanks!

**Review by utkarsh2102 — APPROVED** [review]
Me likey! 👍🏻 
Although you need a rebase or @abhishalya can do that & merge?

**Comment by abhishalya** [comment]
Won't be able to rebase as the PR was created from master. @Marco2018 please rebase if you can.
